### DISCUSSION TITLE
different data models and query condition combinations

### DISCUSSION THREAD

**Discussion by buzai** [prisma/prisma#23452] [Q&A]
### Question

```
{
   where: { postType: 1 },
   include: { 
     user: {
        where: {
           userId: 12
        }
   }
}
```
the postType and userId is AND ？I want two arg is OR query, how to do it

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @buzai 👋 

It's not clear what your question is. Please rephrase the question and fill out the template for us to help you.

**Comment by buzai** [comment]
@ludralph 
I want know how to write those query in prisma?

SELECT * FROM posts LEFT JOIN user ON posts.user_id = user.id
 WHERE postType = 1 AND user.id = 12;

 SELECT * FROM posts LEFT JOIN user ON posts.user_id = user.id
 WHERE postType = 1 OR user.id = 12;

**Reply by ludralph** [reply]
Hi @buzai 

For the first query:
```js
const result = await prisma.post.findMany({
  where: {
    postType: 1,
    author: {
      id: 12
    }
  },
  include: {
    author: true
  }
})
```

For the second query:

```js
const result = await prisma.post.findMany({
  where: {
    OR: [
      { postType: 1 },
      { author: { id: 12 } }
    ]
  },
  include: {
    author: true
  }
})
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
